tron/config/config_parse.py

```diff
@@ -4,6 +4,7 @@
 
 
 WARNING: it is *NOT* safe to delete classes that are being validated (or their attributes) if there are any references to them in DynamoDB! (See DAR-2328)
+NOTE: this means that reverting a change that adds a new attribute is not safe :)
```

Comment:
Maybe we can also add a note that says until TRON-2200 is complete?

Reply:
good idea - done!